bootloader/src/pyi_pythonlib.c

```diff
@@ -769,6 +771,17 @@ pyi_pylib_install_zlibs(ARCHIVE_STATUS *status)
 {
     TOC * ptoc;
 
+    /*
```

Comment:
This code is duplicated from [pyi_launch.c:311](https://github.com/xoviat/pyinstaller/blob/c410aedbdcf5d433edcb5e43c0eb98ed3370874f/bootloader/src/pyi_launch.c#L311-L342). Are the dependencies being extracted twice, now?

Why is it important that extracting the `ARCHIVE_ITEM_DEPENDENCY` files from the other executable is done in the child process and not the parent?